website/snippets/_cloud-environments-info.md

```diff
@@ -34,24 +34,6 @@ Both development and deployment environments have a section called **General Set
 - If you select a current version with `(latest)` in the name, your environment will automatically install the latest stable version of the minor version selected.
 :::
 
-### Git repository caching 
```

Comment:
this section was moved (not deleted) to reorder the four H3 subsections to be alphabetical